optimum/intel/ipex/modeling_base.py

```diff
@@ -153,6 +167,18 @@ def __init__(
         else:
             self._device = torch.device("cpu")
 
+            import numa
+            import psutil
+            n_sockets=get_number_of_sockets()
+            num_cpu_threads_per_process = int(psutil.cpu_count(logical=False) / n_sockets)
+            os.environ["OMP_NUM_THREADS"]=str(num_cpu_threads_per_process)
+            torch.set_num_threads(num_cpu_threads_per_process)
+            numa.set_affinity(0,range(num_cpu_threads_per_process))
+            numa.set_membind([0])
+            print("affinity", numa.get_affinity(0))
+            print("membind", numa.get_membind())
+
```

Comment:
also ,it the OMP_NUM_THREADS, membind is already set by external user. should not override the user's configuration.

also Tensor Parallel case should be considered as well.